brush/brush.go

```diff
@@ -47,18 +47,24 @@ func (b Brush) Use(p *player.Player) {
 	vec := p.Rotation().Vec3().Mul(maxDistance)
 	pos := p.Position().Add(mgl64.Vec3{0, p.EyeHeight()})
 
```

Comment:
This section isn't correct with the world transactions. Use() should probably accept a `*world.EntityHandle` since it's being run from a goroutine, and the rest of the code should be run within the `ExecWorld()` call.